### ISSUE TITLE
Suggestion: supporting materials for aria-roles rule ID

### ISSUE THREAD

**Issue by dboudreau** [dequelabs/axe-core#217]
As per https://github.com/dequelabs/axe-core/issues/216, based on comments received by a client, I suggest we provide a mapping or a comprehensive list of all valid values for each aria role attributes that Axe checks for. That way, it would be easier for our users to understand what an issue is about when the "aria-role" rule ID is triggered. To most, this currently feels like a black box.
